Formula/cassandra.rb

```diff
@@ -13,25 +13,17 @@ class Cassandra < Formula
   end
 
   depends_on "cython" => :build
-  # Due to Python 2 (https://issues.apache.org/jira/browse/CASSANDRA-10190), cassandra 4 will support python3
-  depends_on :macos
-  depends_on "openjdk@8" # cassandra 4 will support Java 11
+  depends_on "python@3.9" => :build
+  depends_on "openjdk@11"
 
-  # Only >=Yosemite has new enough setuptools for successful compile of the below deps.
-  # Python 2 needs setuptools < 45.0.0 (https://github.com/pypa/setuptools/issues/2094)
   resource "setuptools" do
```

Comment:
We shouldn't need this anymore -- Python `venv`s have `setuptools` installed by default.